README.md

```diff
@@ -11,12 +11,16 @@ Current test procedure:
 * make sure you are `cd`'d into your project
 * make sure `git status` is clean in your project
 * Make a **new branch** called `3.0` in which to safely experiment with this tool
-* Type `apos-code-upgrader`
+* Type `apos-code-upgrader lint`
```

Comment:
I am inclined to stick with `lint` because people know what to expect (and what not to expect) from a linter, even if it's idiosyncratic to upgrade-related issues. Happy to discuss more if we need a team naming convo at some point.